nodezator/fontsman/cache.py

```diff
@@ -75,6 +75,7 @@ def __missing__(self, height):
             define at which height the font must be rendered.
         """
         font = get_font(self.font_path, height)
+        self[height] = font
```

Comment:
note: performance fix

Reply:
Very well spotted! This is the actual culprit of the needlessly repeated calls to get_font. I too believe the missing assignment was the result of overlooking this line of code, since I didn't make this mistake in line 48 of the original file.

This is a superb contribution, and something that will no doubt speed up many other services on the app, since most if not all rendered text in the app relies indirectly on this mechanism.

(I accidentally made this comment in the comment regarding the change in line 48 above, when in fact I meant to comment on this change here, in line 78, so I deleted the comment and posted its contents here, where originally meant)